docs/02-for-app-authors/03-appdata-guidelines/01-quality-guidelines.md

```diff
@@ -69,8 +69,62 @@ The summary should ideally be no longer than 35 characters, and must be shorter
 
 ### Not technical
 
-The summary should be understandable by a non-technical person, and not contain any technical terms (e.g. `Chat app written in Rust and GTK4/Libadwaita`). Don't mention the toolkit, programming language, or other implementation details that would not be relevant to a non-developer and explain what the app does for people instead.
+The summary should be understandable by a non-technical person, and not contain any technical terms (e.g. "Chat app written in Rust and GTK4/Libadwaita"). Don't mention the toolkit, programming language, or other implementation details that would not be relevant to a non-developer and explain what the app does for people instead.
```

Comment:
Rolling these back probably was a mistake

Reply:
Yeah not sure how that happened, maybe main changed while I was editing?

Reply:
Yeah, you got a copy of the old file I guess.

Reply:
Should be fixed now.